### PR TITLE
4-directions-endpoint

### PR DESCRIPTION
**Pull request by alexmik95**
Upgraded Ruby version and added Endpoint::Directions and its respective Response.
Now Directions handles only required parameters, origin and destination, as geo-coordinates.
Directions::Response basically takes the first route option for good.
There are many meaningful improvements but for now it is meant to be a basic Google API directions implementation.

### CODE REVIEW

File: google_maps_juice.gemspec [hunk 1]
```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'activesupport', '~> 5.2'
   spec.add_dependency 'excon'
 
-  spec.add_development_dependency 'bundler', '~> 1.16'
+  spec.add_development_dependency 'bundler', '~> 2.0.2'
```
Comment (algodave) [inline google_maps_juice.gemspec]: @alexmik95 Let's stick with bundler `1.17.2`

**Comment by algodave** [comment]
@alexmik95 @safeforge Wdyt of `GoogleMapsJuice::Directions.find()` instead of `GoogleMapsJuice::Directions.directions()` ?

File: spec/integration/directions_spec.rb [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+ROME = '12.496365,41.902783'
+COLOSSEUM = '41.890209,12.492231'
+SAINTPETER = '41.902270,12.457540'
+SIDNEY = '-33.867487,151.206990'
+
+RSpec.describe GoogleMapsJuice::Directions do
+  describe '.directions' do
+    subject { described_class.directions(params) }
+
+    context 'full match on address', vcr: {
```
Comment (algodave) [inline spec/integration/directions_spec.rb]: @alexmik95 Improve message to better describe this context

**Comment by algodave** [comment]
@alexmik95 As a general rule: you can test only the most relevant attributes in integration tests, whereas in unit tests you have to test *all* attributes

**Comment by algodave** [comment]
@alexmik95 Great job with the latest commits! 💯

**Comment by algodave** [comment]
@alexmik95 When you get a chance, please document the Directions endpoint usage in our Readme 🙂

**Comment by algodave** [comment]
@alexmik95 Nice job with the Readme as well 👏 

To me we're ready to merge, wdyt @safeforge ?

**Comment by safeforge** [comment]
LGTM - great job @alexmik95

**Pull request merged**